sql-cli/src/opensearch_sql_cli/opensearchsql_cli.py

```diff
@@ -57,9 +61,21 @@ def __init__(self, clirc_file=None, always_use_pager=False, use_aws_authenticati
         self.multiline_continuation_char = config["main"]["multiline_continuation_char"]
         self.multi_line = config["main"].as_bool("multi_line")
         self.multiline_mode = config["main"].get("multi_line_mode", "src")
+        self.history_file = config["main"]["history_file"]
+        self.log_file = config["main"]["log_file"]
```

Comment:
where is this being used?

Reply:
History file used for history, log file - for logging errors. For example, if config has incorrect path to the history file.

Reply:
Got it - do you need to pass this in anywhere like you do the history file?

Reply:
No actually. Good point, thank you. `self.log_file` removed in f79e5a959.